test/e2e/query_test.go

```diff
@@ -1748,7 +1748,7 @@ func RemoteWrite(ctx context.Context, timeseries []prompb.TimeSeries, addr strin
 	}
 	
 	if res.StatusCode/100 != 2 {
-		return nil, errors.Errorf("request failed with code %s", res.Status)
+		return errors.Errorf("request failed with code %s", res.Status)
 	}
 
 	defer res.Body.Close()
```

Comment:
shouldnt we close body after checking for an error?

Reply:
done